cabby/rvs/item.py

```diff
@@ -41,60 +41,59 @@ class RVSPath:
   `from_file` should the entity be processed from file.
   `instructions` is a basic template that includes the points, pivots and route 
   features.
+  `path_features` is a path features: cardinal directions, intersections, instructions and route geometry.
   """
   start_point: GeoSeries = attr.ib()
   end_point: GeoSeries = attr.ib()
-  route: GeoSeries = attr.ib()
+  route: LineString = attr.ib()
   main_pivot: GeoSeries = attr.ib()
   near_pivot: GeoSeries = attr.ib()
   beyond_pivot: GeoSeries = attr.ib()
   cardinal_direction: Text = attr.ib()
   intersections: int = attr.ib()
   process: bool = attr.ib()
   instructions: Text = attr.ib(init=False)
+  path_features: Dict = attr.ib(init=False)
 
   def __attrs_post_init__(self):
-    if not self.process:
-      return
-    # Creat basic template instruction.
-    if "main_tag" in self.beyond_pivot:
-      avoid_instruction = "If you reached {0}, you have gone too far.".format(
-        self.beyond_pivot['main_tag'])
 
+    if "main_tag" in self.beyond_pivot:
+      avoid_instruction = (
+        f"If you reached {self.beyond_pivot['main_tag']}, you have gone too far.")
```

Comment:
"if you reached" -> "if you reach"

Reply:
Fixed and made sure it is also correct in the real templates (it was).